src/mindMapPreviewWebview.ts

```diff
@@ -47,12 +54,25 @@ class MindMapPreview {
                     .with({ scheme: 'vscode-resource' })}"></script>
             </head>
             <body>
-                <svg id="container"></svg>
+                <svg id="container" preserveAspectRatio="none meet"></svg>
                 <script>
                     const map = markmap.Markmap.create("#container", null, {"t":"heading","d":1,"p":{},"v":"Rendering, please wait..."});
+                    const vscode = acquireVsCodeApi();
                     window.addEventListener('message', event => {
                         const message = event.data;
-                        map.setData(markmap.transform(message));
+                        switch (message.command) {
+                            case "saveSVG": {

```

Comment:
There are 2 coding style problems to fix:
1. Please name the command by little camel-case, such as `saveSvg`
2. Please use the `<verb>-<object>` naming style, for example, `saveSvg`, `renderMarkdown`, `doSomething`.